database/insertData.js

```diff
@@ -172,11 +172,14 @@ const queries = [
                       code: "Q6"
                       index: 7
                       title: "Organisation Category"
-                      elementTypePluginCode: "radioChoice"
+                      elementTypePluginCode: "dropdownChoice"
+                      # Change this to "radioChoice" once we've made the plugin
                       category: QUESTION
                       parameters: {
-                        label: "What category of organisation do you wish to join"
+                        label: "Organisation Type"
+                        description: "Select which type of organisation you belong to."
                         options: ["Manufacturer", "Distributor", "Importer"]
+                        validation: { value: true }
```

Comment:
Why do you need this? I thought this would be the default behaviour

Reply:
Yes, it wasn't -- but it is now.